src/Illuminate/Console/Concerns/HasStubOption.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Illuminate\Console\Concerns;
+
+trait HasStubOption
+{
+    /**
+     * Get the console command arguments.
+     */
+    abstract protected function getOptions(): array;
+
+    /**
+     * Build the class with the given name.
+     *
+     * @param  string  $name
+     * @return string
+     *
+     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
+     */
+    #[\Override]
```

Comment:
Does Override worked on trait?

Reply:
yes, well it works _in_ a trait, but _on_ the class. In other words it requires the class using the trait to have the method being overriden.

Reply:
example here: 

https://php-play.dev/?c=DwfgDgFmBQ0IYCMDOAXATnAxigBJgNnEkjgLICeACnGgKYB2uA3tDmzmAK4L4CWmOAGad62XgHt6OALblStFBHEATABQBKHC3Y6ctTEpwAicrTg5TcCBbMQjAblbsAvtFfR0cXrgoAVDN5aTmwAxADaADoA8gButGhovMq0ALrBHNx8AsKiKBJSsvKKKhpBuuz6hkYQtPj44g7pru4ERCQUAMKExHoAHigMyu1UNAzM6ZxItGTk-l4oju6q9LQA7jNdbRrqALQAfIUKSmrq9kA&v=8.3&f=html